test/compilable/extra-files/dtoh-postscript.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+diff -pu --strip-trailing-cr ${EXTRA_FILES}/${TEST_NAME}.out ${OUTPUT_BASE}.out
+
+rm -f ${OUTPUT_BASE}.out
```

Comment:
Is there really a need for removing this file? Might be useful for debugging when the test failed?

Reply:
The test suite will complain if any files are left over with the shell tests (which I despise). Also this should be made in to a D test rather than a shell one. The less shell tests the better.

Reply:
No it won't. It will only complain if the files aren't placed in the test_results directory.